design/FY2025/NFP-new-air-water-heat-pump.md

```diff
@@ -0,0 +1,1468 @@
+      A39, \field Cooling Energy Input Ratio Function of PLR Curve Name at Speed 3
+        \required-field
+        \type object-list
+        \object-list UnivariateFunctions
+        \note EIRFPLR - Fuel Energy Input Ratio Function of Part Load Ratio(PLR) Curve Name,
+        \note which is a cubic curve or a lookup table. 
+    A40, \field Normalized Cooling Capacity Function of Temperature Curve Name at Speed 4
+        \required-field
+        \type object-list
+        \object-list BivariateFunctions
+        \note: CAPFT - Normalized Capacity Function of Temperature Curve Name,
+        \note which is a biquadratic curve or a lookup table of air and water temperature.
+      A41, \field Cooling Energy Input Ratio Function of Temperature Curve Name at Speed 4
+        \required-field
+        \type object-list
+        \object-list BivariateFunctions
+        \note EIRFT - Fuel Energy Input Ratio Function of Temperature Curve Name,
+        \note which is a biquadratic curve or a lookup table of air and water temperature.
+      A42, \field Cooling Energy Input Ratio Function of PLR Curve Name at Speed 4
+        \required-field
+        \type object-list
+        \object-list UnivariateFunctions
+        \note EIRFPLR - Fuel Energy Input Ratio Function of Part Load Ratio(PLR) Curve Name,
+        \note which is a cubic curve or a lookup table.
+      A43, \field Normalized Cooling Capacity Function of Temperature Curve Name at Speed 5
+        \required-field
+        \type object-list
+        \object-list BivariateFunctions
+        \note: CAPFT - Normalized Capacity Function of Temperature Curve Name,
+        \note which is a biquadratic curve or a lookup table of air and water temperature.
+      A44, \field Cooling Energy Input Ratio Function of Temperature Curve Name at Speed 5
+        \required-field
+        \type object-list
+        \object-list BivariateFunctions
+        \note EIRFT - Fuel Energy Input Ratio Function of Temperature Curve Name,
+        \note which is a biquadratic curve or a lookup table of air and water temperature.
+      A45; \field Cooling Energy Input Ratio Function of PLR Curve Name at Speed 5
+        \required-field
+        \type object-list
+        \object-list UnivariateFunctions
+        \note EIRFPLR - Fuel Energy Input Ratio Function of Part Load Ratio(PLR) Curve Name,
+        \note which is a cubic curve or a lookup table.
+
+### Actuators to enable defrost control through EnergyPlus Python API ###
+
+The following actuators will be added to enable various defrost control strategies using the EnergyPlus API:
+- heat pump operation mode, Mode
+- compressor suction temperature, Tsuc
+- entering water temperature, enteringTemp
+- time since last defrost, timeSinceLast
+- leaving water temperature, leavingTemp
+- time since defrost started, timeSinceStart
+
+Note that ambient air temperature is already an existing EnergyPlus actuator. It could be used in defrost calculation as well.
+
```

Comment:
Please describe the outputs that you are planning to provide. Including timestep output variables, modifications or additions to the tabular outputs, and whether any rated efficiency metrics (such as from AHRI or ISO or other standards organizations) are going to be provided.

Reply:
I will add a section describing the output

Reply:
I added a section listing the output variables

Reply:
Thanks. It would be great to add a tabular output file in the Equipment Summary report. Maybe it could include outputting a calculated rating efficiency equivalent to an AHRI or ISO test procedure (assuming there is one).

Reply:
Thanks! Just added some contents to the "Outputs" section including some info on entries to be added to the Equipment Summary report.